stripe/util.py

```diff
@@ -182,7 +227,7 @@ def convert_to_stripe_object(
 
         return obj
     else:
-        return resp
+        return cast("StripeObject", resp)
```

Comment:
Is there some test-only assert we can add? Would be nice to verify that this is true in tests but without affecting published library runtime.

Reply:
Hmm, I'll noodle on this. I think we have to build our own.